test/functional/feature_taproot.py

```diff
@@ -1230,6 +1230,41 @@ def spenders_taproot_nonstandard():
 
     return spenders
 
+def sample_spenders():
+
+    # Create key(s) for output creation, as well as key and script-spends
+    secs = [generate_privkey() for _ in range(2)]
+    pubs = [compute_xonly_pubkey(sec)[0] for sec in secs]
```

Comment:
Why are two secs and two pubs created if only one is used? Is to show a pattern of generating more than one so other people can add additional tests to this?

Reply:
yep, don't feel strongly on this one but that's probably an expected pattern